merlin/main.py

```diff
@@ -312,8 +319,67 @@ def process_example(args):
         setup_example(args.workflow, args.path)
 
 
+def check_merlin_status(args, spec):
+    """
+    Function to check merlin workers and queues to keep 
+    the allocation alive
+
+    :param `args`: parsed CLI arguments
+    :param `spec`: the parsed spec.yaml
+    """
+    queue_status = router.query_status(
+        args.task_server, spec, args.steps, verbose=False
+    )
+
+    total_jobs = 0
+    total_consumers = 0
+    for name, jobs, consumers in queue_status:
+        total_jobs += jobs
+        total_consumers += consumers
+
+    if total_jobs and total_consumers == 0:
```

Comment:
total_jobs > 0 and total_consumers == 0?
took me a while to parse that. maybe too clever?

Reply:
This is my C optimization showing.

Reply:
oi my slow monkey brain